src/cpp/include/openvino/genai/text2image/clip_text_model_with_projection.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <string>
+
+#include "openvino/genai/visibility.hpp"
+#include "openvino/genai/tokenizer.hpp"
+
+#include "openvino/core/any.hpp"
+#include "openvino/runtime/tensor.hpp"
+#include "openvino/runtime/infer_request.hpp"
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace genai {
+
+class OPENVINO_GENAI_EXPORTS CLIPTextModelWithProjection {
+public:
+    struct Config {
+        size_t max_position_embeddings = 77;
+        size_t hidden_size = 512;
+        size_t num_hidden_layers = 33;
```

Comment:
please, read this value via `read_json_param` in `CLIPTextModelWithProjection::Config::Config`

the same for value in `CLIPTextModel::Config`

Reply:
Done